### DISCUSSION TITLE
Android app crashes server

### DISCUSSION THREAD

**Discussion by yonas** [dani-garcia/vaultwarden#4934] [Q&A]
### Vaultwarden Support String

TODO: Will add in a few minutes.

### Vaultwarden Build Version

1.30.0

### Deployment method

OS Package (apt, yum/dnf, pacman, apk, nix, ...)

### Reverse Proxy

None

### Host/Server Operating System

Other

### Operating System Version

FreeBSD 14

### Clients

Android

### Client Version

Latest Android

### Steps To Reproduce

1. Start vaultwarden
2. Turn on mobile phone with bitwarden already installed
3. Discover vaultwarden crashed with an error message in logs

### Expected Result

No crash.

### Actual Result

Server crashed.

### Logs

```text
Sep 7 11:41:30 server1 vaultwarden[9748]: [2024-09-07 11:41:30.875][vaultwarden::api::core::two_factor][INFO] User yonas@mail.lan did not complete a 2FA login within the configured time limit. IP: 127.0.0.1
Sep 7 11:41:30 server1 vaultwarden[9748]: [2024-09-07 11:41:30.886][vaultwarden::mail][ERROR] SMTP error: Connection error: Connection refused (os error 61)
Sep 7 11:41:30 server1 vaultwarden[9748]: [2024-09-07 11:41:30.890][panic][ERROR] thread 'tokio-runtime-worker' panicked at 'Error sending incomplete 2FA email: SMTP error: Connection error: Connection refused (os error 61)': src/api/core/two_factor/mod.rs:205
```

**Comment by yonas** [comment]
> Error sending incomplete 2FA email: SMTP error: Connection error: Connection refused

Can I disable this email from needing to be sent.

**Reply by stefan0xC** [reply]
You can set `INCOMPLETE_2FA_TIME_LIMIT=0` to disable the check.
https://github.com/dani-garcia/vaultwarden/blob/66baa5e7d844a455ffe18a89e9cb8521a013af14/.env.template#L214-L219

**Comment by stefan0xC** [comment]
This is a duplicate of #4528 and has already been fixed by  #4792 which is in the latest release (1.32.0). So you should [update your Vaultwarden](https://github.com/dani-garcia/vaultwarden/issues/1180).
